mlflow/utils/databricks_utils.py

```diff
@@ -187,6 +187,14 @@ def is_in_databricks_model_serving_environment():
     return val.lower() == "true"
 
 
+def enable_mlflow_tracing() -> bool:
```

Comment:
```suggestion
def enable_mlflow_tracing_in_model_serving() -> bool:
```

We may extend this env var outside model serving, but maybe it's better to name accurately until then?

Reply:
Can we use `is_enabled_...`? `enable_...` sounds like we mutate something (e.g. set the flag to True).

Reply:
I'm fine with `enable_mlflow_tracing` or `enable_mlflow_tracing_in_model_serving` if this must align with databricks.

Reply:
Updated to is_mlflow_tracing_enabled_in_model_serving